### PR TITLE
added small example for using crow

### PR DESCRIPTION
**Pull request by StephanKa**

### CODE REVIEW

File: src/CMakeLists.txt [hunk 1]
```diff
@@ -13,6 +14,12 @@ IF(CPP_STARTER_USE_BOOST_BEAST)
     ADD_SUBDIRECTORY(boost.beast)
 ENDIF()
 
+# Crow
+IF(CPP_STARTER_USE_BOOST_BEAST)
```
Comment (arnemertz) [inline src/CMakeLists.txt]: copy paste error?
`IF(CPP_STARTER_USE_CROW)`

File: src/crow/main.cpp [hunk 2]
```diff
@@ -0,0 +1,51 @@
...
+};
+
+int main()
+{
+  constexpr uint16_t port = 8080;
+  std::vector<int> values;
+  crow::SimpleApp app;
+
+  CROW_ROUTE(app, "/")
+  ([]() {
```
Comment (arnemertz) [inline src/crow/main.cpp:22]: in the other lambdas, you omitted the empty parentheses ;-)

File: src/crow/main.cpp [hunk 3]
```diff
@@ -0,0 +1,51 @@
...
+    return "Hello world";
+  });
+
+  CROW_ROUTE(app, "/json")
+  ([] {
+    crow::json::wvalue returnValue({ { "message", "Hello, World!" } });
+    returnValue["message2"] = "Hello, World.. Again!";
```
Comment (arnemertz) [inline src/crow/main.cpp]: double period on purpose?

**Review by arnemertz — CHANGES_REQUESTED** [review]

File: src/CMakeLists.txt (hunk 1 above)
Comment (StephanKa) [reply src/CMakeLists.txt]: yep :( -> fixed

File: src/crow/main.cpp (hunk 2 above)
Comment (StephanKa) [reply src/crow/main.cpp:22]: fixed :)

File: src/crow/main.cpp (hunk 3 above)
Comment (StephanKa) [reply src/crow/main.cpp]: reduced code -> done

File: src/crow/main.cpp (hunk 2 above)
Comment (arnemertz) [reply src/crow/main.cpp:22]: parentheses here...

File: src/crow/main.cpp [hunk 4]
```diff
@@ -0,0 +1,49 @@
...
+  std::vector<int> values;
+  crow::SimpleApp app;
+
+  CROW_ROUTE(app, "/")
+  ([]() {
+    return "Hello world";
+  });
+
+  CROW_ROUTE(app, "/json")
+  ([] {
```
Comment (arnemertz) [inline src/crow/main.cpp:27]: no parentheses here...

File: src/crow/main.cpp [hunk 5]
```diff
@@ -0,0 +1,49 @@
...
+    return crow::json::wvalue({ { "message", "Hello, World!" }, {"message2", "Hello, World.. Again!"} });
+  });
+
+  CROW_ROUTE(app, "/custom")
+  ([]() {
```
Comment (arnemertz) [inline src/crow/main.cpp:32]: parentheses here...

**Review by arnemertz — APPROVED** [review]

**Pull request merged**